backend/tests/Notifo.Domain.Tests/Integrations/Discord/DiscordTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Notifo.Domain.Integrations.Discord;
+public sealed class DiscordTests
+{
+    [Fact]
+    public void Should_get_integration()
+    {
+        var client = CreateClient();
+        Assert.NotNull(client.System);
+    }
+
+    [Fact]
+    public async Task Should_send_simple_message()
+    {
+        var client = CreateClient();
+
+        var message = new MessagingMessage
+        {
+            Text = "Test message",
```

Comment:
You never test your new Body.

Reply:
If you mean the `Body` field, I'm actually testing it in a test below (`Send_full_message`). Also, I noticed that I forgot the "Should_" prefix. Fixing that...